utils/shared/src/main/scala/fastparse/Utils.scala

```diff
@@ -82,17 +105,40 @@ object Utils {
       }
       res.mkString
     }
-    def compute(chars: Seq[Char]) = {
-      val first = chars.min
-      val last = chars.max
+  }
+
+  trait ElemHelper[Elem] {
```

Comment:
Possible to move this into the same file as all the `ParserHelper` stuff? I'd like to keep all the type-specific helpers in one file so we can quickly skim over them and see what's there